packages/spektate/src/pipeline/AzureDevOpsPipeline.ts

```diff
@@ -34,7 +34,9 @@ export class AzureDevOpsPipeline implements IPipeline {
     this.pipelineAccessToken = pipelineAccessToken;
   }
 
-  public async getListOfBuilds(buildIds?: Set<string>) {
+  public async getListOfBuilds(
+    buildIds?: Set<string>
+  ): Promise<{ [id: string]: IBuild }> {
```

Comment:
maybe this can use `IBuilds` in `Pipeline.ts`?

Reply:
accept